scripts/devenv-builder/configure-vm.sh

```diff
@@ -63,7 +76,9 @@ sudo subscription-manager config --rhsm.manage_repos=1
 sudo subscription-manager repos \
     --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
 #    --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
-sudo dnf localinstall -y ~/microshift/_output/rpmbuild/RPMS/*/*.rpm
+if $BUILD_AND_INSTALL ; then
+    sudo dnf localinstall -y ~/microshift/_output/rpmbuild/RPMS/*/*.rpm
+fi
 
 sudo cp -f ${OCP_PULL_SECRET} /etc/crio/openshift-pull-secret
```

Comment:
Copy of the pull secret should be conditional on BUILD_AND_INSTALL as well. /etc/crio does not necessarily  exist on a build system and copy of the pull secret into the ISO is already handled by the build.sh script.